sdks/java/core/src/main/java/org/apache/beam/sdk/schemas/transforms/Cast.java

```diff
@@ -300,7 +300,7 @@ public void verifyCompatibility(Schema inputSchema) {
 
                   @ProcessElement
                   public void process(
-                      @FieldAccess("filterFields") Row input, OutputReceiver<Row> r) {
+                      @FieldAccess("filterFields") @Element Row input, OutputReceiver<Row> r) {
```

Comment:
Why both annotations?

Reply:
After this PR, FieldAccess no longer replaces Element. However the next PR will remove the need for Element (requires more refactoring)

Reply:
With this PR, elements always need Element annotations. This restriction is removed in the next PR.